app/src/behaviors/behavior_hold_tap.c

```diff
@@ -699,6 +716,9 @@ static int behavior_hold_tap_init(const struct device *dev) {
         .global_quick_tap = DT_INST_PROP(n, global_quick_tap),                                     \
         .flavor = DT_ENUM_IDX(DT_DRV_INST(n), flavor),                                             \
         .retro_tap = DT_INST_PROP(n, retro_tap),                                                   \
+        .retro_tap_behavior = DT_INST_PROP(n, retro_tap_behavior),                                 \
```

Comment:
instead of this, just use `.retro_tap_behavior_dev = DT_LABEL(DT_INST_PHANDLE_BY_IDX(n, bindings, 2)),` (although we need to figure out how to make it fall back to the tap binding)